# Simplified Chinese translation for gdebi
# Copyright (c) 2007 Rosetta Contributors and Canonical Ltd 2007
# This file is distributed under the same license as the gdebi package.
# rainofchaos <rainofchaos@gmail.com>, 2007.
# rainofchaos <rainofchaos@gmail.com>, 2008.
#
#
msgid ""
msgstr ""
"Project-Id-Version: gdebi\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2016-12-08 16:02+0000\n"
"PO-Revision-Date: 2011-11-29 04:41+0000\n"
"Last-Translator: Wylmer Wang <Unknown>\n"
"Language-Team: Simplified Chinese <zh_CN@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2014-03-12 19:04+0000\n"
"X-Generator: Launchpad (build 16963)\n"

#: ../gdebi:50
#, c-format
msgid ""
"usage: %prog [options] filename\n"
"For a graphical version run gdebi-gtk\n"
msgstr ""
"用法：%prog [选项] 文件名\n"
"要使用图形化版本，可以运行 gdebi-gtk\n"

#: ../gdebi:56 ../gdebi-gtk:54
msgid "Run non-interactive (dangerous!)"
msgstr "以非交互式运行（危险！）"

#: ../gdebi:60
msgid "Set an APT configuration option"
msgstr "设置一个 APT 配置选项"

#: ../gdebi:64
msgid "Do not show progress information"
msgstr "不显示进度信息"

#: ../gdebi:68
msgid "Simulate only and print a apt-get install compatible line to stderr"
msgstr ""
"仅模拟执行并向标准错误（stderr）输出一行与 apt-get install 兼容的信息。"

#: ../gdebi:70
msgid "Use alternative root dir"
msgstr "使用另外的 root 文件夹"

#: ../gdebi:78
#, c-format
msgid "gdebi error, file not found: %s\n"
msgstr "gdebi 错误，找不到文件：%s\n"

#: ../gdebi:97 ../gdebi:106
msgid "Need to be root to install packages"
msgstr "需要以 root 权限安装软件包"

#: ../gdebi:108 ../GDebi/GDebiCli.py:154
msgid "Do you want to install the software package? [y/N]:"
msgstr "您是否想安装这个软件包？[y/N]:"

#: ../gdebi-gtk:57
msgid "Auto close when the install is finished"
msgstr "安装完成后自动关闭"

#: ../gdebi-gtk:59
msgid "Use alternative datadir"
msgstr "使用另外的数据路径"

#: ../gdebi-gtk:62
msgid "Remove package"
msgstr "移除软件包"

#: ../gdebi-gtk:81
msgid "Software index is broken"
msgstr "软件索引已损坏"

#: ../gdebi-gtk:82
msgid ""
"This is a major failure of your software management system. Please check for "
"broken packages with synaptic, check the file permissions and correctness of "
"the file '/etc/apt/sources.list' and reload the software information with: "
"'sudo apt-get update' and 'sudo apt-get install -f'."
msgstr ""
"这是您软件管理系统的重大失败。请使用新立得检查损坏的软件包，检查文件权限和 '/"
"etc/apt/sources.list' 文件的正确性并使用 'sudo apt-get update' 和 'sudo apt-"
"get install -f' 重新载入软件信息。"

#: ../data/gdebi.ui.h:1 ../data/gdebi.desktop.in.h:2
msgid "Package Installer"
msgstr "软件包安装程序"

#: ../data/gdebi.ui.h:2
msgid "<b><big>   </big></b>"
msgstr "<b><big> </big></b>"

#: ../data/gdebi.ui.h:3
msgid "_Details"
msgstr "细节(_D)"

#: ../data/gdebi.ui.h:4 ../GDebi/GDebiKDE.py:155
msgid "Description"
msgstr "描述"

#: ../data/gdebi.ui.h:5 ../GDebi/GDebiKDE.py:161
msgid "<b>Version:</b>"
msgstr "<b>版本：</b>"

#: ../data/gdebi.ui.h:6 ../GDebi/GDebiKDE.py:162
msgid "<b>Maintainer:</b>"
msgstr "<b>维护者：</b>"

#: ../data/gdebi.ui.h:7 ../GDebi/GDebiKDE.py:163
msgid "<b>Priority:</b>"
msgstr "<b>优先级：</b>"

#: ../data/gdebi.ui.h:8 ../GDebi/GDebiKDE.py:164
msgid "<b>Section:</b>"
msgstr "<b>组别：</b>"

#: ../data/gdebi.ui.h:9 ../GDebi/GDebiKDE.py:165
msgid "<b>Size:</b>"
msgstr "<b>大小：</b>"

#: ../data/gdebi.ui.h:10
msgid "  "
msgstr "  "

#: ../data/gdebi.ui.h:11 ../GDebi/GDebiKDE.py:154 ../GDebi/GDebiKDE.py:156
#: ../GDebi/GDebiKDE.py:318
msgid "Details"
msgstr "细节"

#: ../data/gdebi.ui.h:12
msgid "Included files"
msgstr "包含的文件"

#: ../data/gdebi.ui.h:13
msgid "Lintian output"
msgstr ""

#. img = Gtk.Image()
#. img.set_from_stock(Gtk.STOCK_APPLY,Gtk.IconSize.BUTTON)
#. self.button_install.set_image(img)
#: ../data/gdebi.ui.h:14 ../GDebi/GDebiGtk.py:336 ../GDebi/GDebiGtk.py:394
msgid "_Install Package"
msgstr "安装软件包(_I)"

#: ../data/gdebi.ui.h:15 ../data/gdebi.desktop.in.h:3
msgid "Install and view software packages"
msgstr "安装和查看软件包"

#: ../data/gdebi.ui.h:16
msgid "GPL, see /usr/share/common-licenses/GPL"
msgstr "GPL, 参见 /usr/share/common-licenses/GPL"

#: ../data/gdebi.ui.h:17
msgid "Terminal"
msgstr "终端"

#: ../data/gdebi.ui.h:18
msgid "Automatically close after the changes have been successfully applied"
msgstr "在成功应用更改后自动关闭"

#: ../data/gdebi.ui.h:19 ../GDebi/GDebiKDE.py:317
msgid "<b>To install the following changes are required:</b>"
msgstr "<b>要进行安装，需要做以下改变：</b>"

#: ../data/gdebi.desktop.in.h:1
msgid "GDebi Package Installer"
msgstr "GDebi 软件包安装程序"

#: ../data/gdebi.desktop.in.h:4
msgid "package;apt;dpkg;install"
msgstr ""

#. Translators: it's for missing entries in the deb package,
#. e.g. a missing "Maintainer" field
#: ../GDebi/DebPackage.py:38
#, python-format
msgid "%s is not available"
msgstr "%s 不可用"

#: ../GDebi/DebPackage.py:52
msgid "Click packages can currently only be inspected with this tool"
msgstr ""

#: ../GDebi/GDebiGtk.py:95
msgid "Copy selected text"
msgstr "复制所选文字"

#: ../GDebi/GDebiGtk.py:106
msgid "Loading..."
msgstr "正在加载..."

#: ../GDebi/GDebiGtk.py:181
msgid "Can not download as root"
msgstr "无法以 root 身份下载"

#: ../GDebi/GDebiGtk.py:182
msgid ""
"Remote packages can not be downloaded when running as root. Please try again "
"as a normal user."
msgstr "以 root 身份运行时无法下载远程软件包。请以普通用户身份重试。"

#: ../GDebi/GDebiGtk.py:195
msgid "Downloading package"
msgstr "正在下载软件包"

#: ../GDebi/GDebiGtk.py:202
msgid "Download failed"
msgstr "下载失败"

#: ../GDebi/GDebiGtk.py:203
#, python-format
msgid "Downloading the package failed: file '%s' '%s'"
msgstr "软件包下载失败：文件 '%s' '%s'"

#. set window title
#. set name
#: ../GDebi/GDebiGtk.py:251 ../GDebi/GDebiKDE.py:205
#, python-format
msgid "Package Installer - %s"
msgstr "软件包安装程序 - %s"

#: ../GDebi/GDebiGtk.py:307
msgid "Package control data"
msgstr "软件包控制数据"

#: ../GDebi/GDebiGtk.py:310
msgid "Upstream data"
msgstr "上游数据"

#: ../GDebi/GDebiGtk.py:316
msgid "Error reading filelist"
msgstr "读取文件列表出错"

#: ../GDebi/GDebiGtk.py:332
msgid "Error: "
msgstr "错误： "

#: ../GDebi/GDebiGtk.py:347
msgid "Error: no longer provides "
msgstr "错误：已不再提供 "

#: ../GDebi/GDebiGtk.py:364
msgid "Same version is already installed"
msgstr "已安装相同版本"

#: ../GDebi/GDebiGtk.py:365
msgid "_Reinstall Package"
msgstr "重新安装软件包(_R)"

#: ../GDebi/GDebiGtk.py:403
msgid ""
"No lintian available.\n"
"Please install using sudo apt-get install lintian"
msgstr ""

#: ../GDebi/GDebiGtk.py:406
msgid "Running lintian..."
msgstr ""

#: ../GDebi/GDebiGtk.py:428
#, python-format
msgid ""
"\n"
"Lintian finished with exit status %s"
msgstr ""

#: ../GDebi/GDebiGtk.py:459
msgid "Selection is a directory"
msgstr "选择的是一个目录"

#: ../GDebi/GDebiGtk.py:464 ../GDebi/GDebiGtk.py:470
#, python-format
msgid "Error reading file content '%s'"
msgstr "读取文件内容“%s”时出错"

#: ../GDebi/GDebiGtk.py:475
msgid "File content can not be extracted"
msgstr "无法提取文件内容"

#: ../GDebi/GDebiGtk.py:486
#, python-format
msgid "<b>To be removed: %s</b>"
msgstr "<b>将要删除：%s</b>"

#: ../GDebi/GDebiGtk.py:488 ../GDebi/GDebiKDE.py:311
#, python-format
msgid "To be installed: %s"
msgstr "将要安装：%s"

#: ../GDebi/GDebiGtk.py:503
msgid "Open Software Package"
msgstr "打开软件包"

#: ../GDebi/GDebiGtk.py:508
msgid "Software packages"
msgstr "软件包"

#: ../GDebi/GDebiGtk.py:557
msgid "Dependency problems"
msgstr "依赖问题"

#: ../GDebi/GDebiGtk.py:558
#, python-format
msgid "One or more packages are required by %s, it cannot be removed."
msgstr "%s 依赖一个或多个软件包，不能移除它。"

#: ../GDebi/GDebiGtk.py:566
msgid "File not found"
msgstr "文件未找到"

#: ../GDebi/GDebiGtk.py:567
msgid "You tried to install a file that does not (or no longer) exist. "
msgstr "您试图安装一个不存在(或不再存在)的文件。 "

#: ../GDebi/GDebiGtk.py:578
msgid "Installing package file..."
msgstr "正在安装软件包文件..."

#: ../GDebi/GDebiGtk.py:580
msgid "Removing package..."
msgstr "正在移除软件包..."

#: ../GDebi/GDebiGtk.py:583
msgid "Install unauthenticated software?"
msgstr "安装未认证的软件？"

#: ../GDebi/GDebiGtk.py:584
msgid ""
"Malicious software can damage your data and take control of your system.\n"
"\n"
"The packages below are not authenticated and could therefore be of malicious "
"nature."
msgstr ""
"恶意软件会破坏您的数据并控制您的系统。\n"
"\n"
"以下的软件包未经过验证，可能会具有恶意性质。"

#: ../GDebi/GDebiGtk.py:612
msgid "You need to grant administrative rights to install software"
msgstr "你需要获取管理员权限以安装软件"

#: ../GDebi/GDebiGtk.py:613
msgid ""
"\n"
"It is a possible security risk to install packages files manually.\n"
"Install software from trustworthy software distributors only.\n"
msgstr ""
"\n"
"手工安装软件包是一种潜在的安全风险。\n"
"只从值得信任的软件发行者的渠道安装软件。\n"

#: ../GDebi/GDebiGtk.py:618
msgid "You need to grant administrative rights to remove software"
msgstr "您需要获得管理员权限来移除软件"

#: ../GDebi/GDebiGtk.py:619
msgid "It is a possible risk to remove packages."
msgstr "移除软件包有潜在的风险。"

#: ../GDebi/GDebiGtk.py:641 ../GDebi/GDebiGtk.py:706
msgid "Failed to install package file"
msgstr "安装软件包文件失败"

#: ../GDebi/GDebiGtk.py:643
msgid "Failed to remove package"
msgstr "移除软件包失败"

#. errMsg = "%s" % msg
#: ../GDebi/GDebiGtk.py:688 ../GDebi/GDebiKDE.py:366
msgid "Could not download all required files"
msgstr "无法下载全部必需的文件"

#: ../GDebi/GDebiGtk.py:689
#, fuzzy
msgid ""
"Please check your internet connection or installation medium, and make sure "
"your APT cache is up-to-date."
msgstr "请检查您的互联网连接或安装介质。"

#: ../GDebi/GDebiGtk.py:695 ../GDebi/GDebiKDE.py:373
msgid "Could not install all dependencies"
msgstr "无法安装所有依赖软件包"

#: ../GDebi/GDebiGtk.py:696 ../GDebi/GDebiKDE.py:374
msgid ""
"Usually this is related to an error of the software distributor. See the "
"terminal window for more details."
msgstr "通常这与软件发行者的一个错误有关。查看终端窗口输出获取更多细节。"

#: ../GDebi/GDebiGtk.py:712
#, python-format
msgid "Installing %s"
msgstr "正在安装 %s"

#: ../GDebi/GDebiGtk.py:715
#, python-format
msgid "Removing %s"
msgstr "正在移除 %s"

#. self.label_action.set_markup("<b><big>"+_("Package installed")+"</big></b>")
#. show the button
#. self.button_deb_install_close.set_sensitive(True)
#. self.button_deb_install_close.grab_default()
#: ../GDebi/GDebiGtk.py:737 ../GDebi/GDebiKDE.py:393
msgid "Installation finished"
msgstr "安装完成"

#: ../GDebi/GDebiGtk.py:739
msgid "Removal finished"
msgstr "移除结束"

#: ../GDebi/GDebiGtk.py:742
#, python-format
msgid "Package '%s' was installed"
msgstr "软件包'%s'已安装"

#: ../GDebi/GDebiGtk.py:744
#, python-format
msgid "Package '%s' was removed"
msgstr "软件包“%s”已移除"

#: ../GDebi/GDebiGtk.py:747 ../GDebi/GDebiKDE.py:397
#, python-format
msgid "Failed to install package '%s'"
msgstr "安装软件包'%s'失败"

#: ../GDebi/GDebiGtk.py:750
#, python-format
msgid "Failed to remove package '%s'"
msgstr "移除软件包“%s”失败"

#: ../GDebi/GDebiGtk.py:754
msgid "Installation complete"
msgstr "安装完成"

#: ../GDebi/GDebiGtk.py:756
msgid "Removal complete"
msgstr "移除完成"

#: ../GDebi/GDebiGtk.py:764 ../GDebi/GDebiKDE.py:407
msgid "Failed to completely install all dependencies"
msgstr "未能完整地安装所有依赖软件包"

#: ../GDebi/GDebiGtk.py:766
msgid "Failed to completely remove package"
msgstr "未能完全移除软件包"

#: ../GDebi/GDebiGtk.py:767 ../GDebi/GDebiKDE.py:408
msgid "To fix this run 'sudo apt-get install -f' in a terminal window."
msgstr "要修复，在终端窗口中运行 'sudo apt-get install -f' 。"

#. ui
#: ../GDebi/GDebiGtk.py:885 ../GDebi/KDEAptDialogs.py:69
#, python-format
msgid "Installing '%s'..."
msgstr "正在安装 '%s'..."

#: ../GDebi/GDebiGtk.py:888
#, python-format
msgid "Removing '%s'..."
msgstr "正在移除“%s”..."

#: ../GDebi/GDebiGtk.py:998 ../GDebi/KDEAptDialogs.py:135
msgid "Installing dependencies..."
msgstr "正在安装依赖软件包..."

#: ../GDebi/GDebiGtk.py:1043 ../GDebi/KDEAptDialogs.py:198
#: ../GDebi/KDEAptDialogs.py:208 ../GDebi/KDEAptDialogs.py:210
msgid "Downloading additional package files..."
msgstr "正在下载额外的软件包文件..."

#: ../GDebi/GDebiGtk.py:1052 ../GDebi/KDEAptDialogs.py:208
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "文件 %s/%s 速度： %sB/s"

#: ../GDebi/GDebiGtk.py:1054 ../GDebi/KDEAptDialogs.py:210
#, python-format
msgid "File %s of %s"
msgstr "文件 %s/%s"

#. print "mediaChange %s %s" % (medium, drive)
#: ../GDebi/GDebiGtk.py:1061 ../GDebi/KDEAptDialogs.py:216
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "请将 '%s' 插入驱动器 '%s' 中"

#: ../GDebi/GDebiCli.py:59
msgid "Configuration items must be specified with a =<value>\n"
msgstr "配置项必须以 =<值> 的方式指定\n"

#: ../GDebi/GDebiCli.py:65
#, python-format
msgid "Couldn't set APT option %s to %s\n"
msgstr "无法将 APT 选项 %s 设为 %s\n"

#: ../GDebi/GDebiCli.py:78
#, python-format
msgid "Unknown package type '%s', exiting\n"
msgstr "未知软件包类型 '%s'，退出\n"

#: ../GDebi/GDebiCli.py:82
msgid "Failed to open the software package\n"
msgstr "打开软件包失败\n"

#: ../GDebi/GDebiCli.py:83
msgid ""
"The package might be corrupted or you are not allowed to open the file. "
"Check the permissions of the file.\n"
msgstr "软件包可能已被破坏，或者不允许您打开此文件。检查文件的权限。\n"

#: ../GDebi/GDebiCli.py:89
msgid "This package is uninstallable\n"
msgstr "此软件包不可安装\n"

#: ../GDebi/GDebiCli.py:98 ../GDebi/GDebiKDE.py:241
msgid "No description is available"
msgstr "没有可用的描述"

#: ../GDebi/GDebiCli.py:108
msgid "The following packages are UNAUTHENTICATED: "
msgstr "下列软件包未经认证： "

#: ../GDebi/GDebiCli.py:112
msgid "Requires the REMOVAL of the following packages: "
msgstr "需要 移除 以下软件包： "

#: ../GDebi/GDebiCli.py:117
msgid "Requires the installation of the following packages: "
msgstr "需要安装以下软件包： "

#: ../GDebi/GDebiCli.py:132
#, python-format
msgid "Error during install: '%s'"
msgstr "安装时出错： '%s'"

#. first, we load all the default descriptions -- pyuic doesn't use
#. gettext as default (FIXME, copy code from language-selector)
#: ../GDebi/GDebiKDE.py:152
msgid "Package:"
msgstr "软件包："

#: ../GDebi/GDebiKDE.py:153
msgid "Status:"
msgstr "状态:"

#: ../GDebi/GDebiKDE.py:157
msgid "Included Files"
msgstr "包含文件"

#: ../GDebi/GDebiKDE.py:159 ../GDebi/GDebiKDE.py:299
msgid "&Install Package"
msgstr "安装软件包(&I)"

#: ../GDebi/GDebiKDE.py:160
msgid "&Download Package"
msgstr "下载软件包(&D)"

#: ../GDebi/GDebiKDE.py:185
msgid "The package file does not exist"
msgstr "软件包文件不存在"

#: ../GDebi/GDebiKDE.py:186
msgid ""
"A nonexistent file has been selected for installation. Please select an "
"existing .deb package file."
msgstr "选择了一个不存在的文件来安装。请选择一个存在的 .deb 软件包文件。"

#. self.textLabel1_3_2.setText(_("Same version is already installed"))
#: ../GDebi/GDebiKDE.py:267
msgid "&Reinstall Package"
msgstr "重新安装软件包(&R)"

#: ../GDebi/GDebiKDE.py:302
msgid "Re&install Package"
msgstr "重新安装软件包(&I)"

#: ../GDebi/GDebiKDE.py:313
#, python-format
msgid "To be removed: %s"
msgstr "要移除的： %s"

#: ../GDebi/GDebiKDE.py:367
#, fuzzy
msgid ""
"Please check your internet connection or installation medium, or make sure "
"your APT cache is up-to-date."
msgstr "请检查您的互联网连接或安装介质。"

#: ../GDebi/GDebiKDE.py:395
#, python-format
msgid "<b>Package '%s' was installed</b>"
msgstr ""

#: ../GDebi/GDebiKDE.py:421
msgid "Installing"
msgstr ""

#: ../GDebi/GDebiCommon.py:78
msgid "Broken dependencies"
msgstr "损坏的依赖关系"

#: ../GDebi/GDebiCommon.py:79
msgid ""
"Your system has broken dependencies. This application can not continue until "
"this is fixed. To fix it run 'gksudo synaptic' or 'sudo apt-get install -f' "
"in a terminal window."
msgstr ""
"您的系统中存在损坏的依赖关系。不修复它此程序无法继续。要修复，在终端窗口中运"
"行 'gksudo synaptic' 或 'sudo apt-get install -f' 。"

#: ../GDebi/GDebiCommon.py:100
#, python-format
msgid "'%s' is not a Debian package"
msgstr "'%s' 不是一个 Debian 软件包"

#: ../GDebi/GDebiCommon.py:101
#, python-format
msgid ""
"The MIME type of this file is '%s' and can not be installed on this system."
msgstr "此文件的 MIME 类型为 '%s' ，不能安装到系统上。"

#: ../GDebi/GDebiCommon.py:105
#, python-format
msgid "Could not open '%s'"
msgstr "打不开 '%s'"

#: ../GDebi/GDebiCommon.py:106
msgid ""
"The package might be corrupted or you are not allowed to open the file. "
"Check the permissions of the file."
msgstr "软件包可能已损坏或不允许您打开此文件。检查文件的权限。"

#: ../GDebi/GDebiCommon.py:128
msgid "Same version is available in a software channel"
msgstr "软件仓库中有相同版本"

#: ../GDebi/GDebiCommon.py:129
msgid "You are recommended to install the software from the channel instead."
msgstr "建议您从软件仓库中安装这个软件。"

#: ../GDebi/GDebiCommon.py:133
msgid "An older version is available in a software channel"
msgstr "软件渠道中有较旧的可用版本。"

#: ../GDebi/GDebiCommon.py:134
msgid ""
"Generally you are recommended to install the version from the software "
"channel, since it is usually better supported."
msgstr "您最好还是从软件仓库中安装这个软件，因为它通常有更好的支持。"

#: ../GDebi/GDebiCommon.py:139
msgid "A later version is available in a software channel"
msgstr "软件仓库中有更新的版本"

#: ../GDebi/GDebiCommon.py:141
msgid ""
"You are strongly advised to install the version from the software channel, "
"since it is usually better supported."
msgstr "强烈建议您从软件仓库中安装这个软件，因为它通常有更好的支持。"

#: ../GDebi/GDebiCommon.py:183
msgid "All dependencies are satisfied"
msgstr "所有依赖关系已满足"

#. FIXME: use ngettext here
#: ../GDebi/GDebiCommon.py:186
#, python-format
msgid "Requires the <b>removal</b> of %s packages\n"
msgstr "需要 the <b>移除</b>  %s 个软件包\n"

#: ../GDebi/GDebiCommon.py:188
#, python-format
msgid "Requires the installation of %s packages"
msgstr "需要安装 %s 个软件包"

#: ../GDebi/GDebiCommon.py:196 ../GDebi/GDebiCommon.py:214
msgid "Only one software management tool is allowed to run at the same time"
msgstr "同时只允许运行一个软件管理工具"

#: ../GDebi/GDebiCommon.py:198 ../GDebi/GDebiCommon.py:216
msgid ""
"Please close the other application e.g. 'Update Manager', 'aptitude' or "
"'Synaptic' first."
msgstr "请先关闭其他应用程序如 'Update Manager'、 'aptitude' 或 'Synaptic' 。"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../GDebi/KDEAptDialogs.py:218
msgid "Media Change"
msgstr "媒介改变"

#~ msgid "_Download Package"
#~ msgstr "下载软件包(_D)"

#~ msgid "_Remove Package"
#~ msgstr "移除软件包(_R)"

#~ msgid "_File"
#~ msgstr "文件(_F)"

#~ msgid "_Open…"
#~ msgstr "打开(_O)..."

#~ msgid "_Refresh"
#~ msgstr "刷新(_R)"

#~ msgid "_Edit"
#~ msgstr "编辑(_E)"

#~ msgid "_Help"
#~ msgstr "帮助(_H)"

#~ msgid "Description:"
#~ msgstr "描述："

#~ msgid "gdebi"
#~ msgstr "gdebi"

#~ msgid "Wrong architecture '%s'"
#~ msgstr "错误的架构 '%s'"

#~ msgid "translator-credits"
#~ msgstr ""
#~ "Launchpad Contributions:\n"
#~ "  Careone https://launchpad.net/~zzbusagain\n"
#~ "  Colin Zhao https://launchpad.net/~colin-shzsc\n"
#~ "  Liu Qishuai https://launchpad.net/~lqs\n"
#~ "  Qiu Haoyu https://launchpad.net/~timothyqiu\n"
#~ "  Tao Wei https://launchpad.net/~weitao1979\n"
#~ "  Wylmer Wang https://launchpad.net/~wantinghard\n"
#~ "  Xhacker Liu https://launchpad.net/~xhacker\n"
#~ "  Yunkwan https://launchpad.net/~chanyunkwan0217\n"
#~ "  ZhengPeng Hou https://launchpad.net/~zhengpeng-hou\n"
#~ "  kEEg https://launchpad.net/~jimcheng\n"
#~ "  stone_unix https://launchpad.net/~gaoghy\n"
#~ "  zhangmiao https://launchpad.net/~mymzhang\n"
#~ "  冯超 https://launchpad.net/~rainofchaos"

#~ msgid "Y"
#~ msgstr "Y"

#~ msgid "Open..."
#~ msgstr "打开..."

#~ msgid "Software package"
#~ msgstr "软件包"

#~ msgid "Dependency is not satisfiable: %s\n"
#~ msgstr "依赖关系不满足： %s\n"

#~ msgid "Conflicts with the installed package '%s'\n"
#~ msgstr "与已安装的软件包 '%s' 冲突\n"

#~ msgid "<b>"
#~ msgstr "<b>"

#~ msgid "Analysing dependencies"
#~ msgstr "正在分析依赖关系"

#~ msgid "No Architecture field in the package"
#~ msgstr "软件包中没有 架构 字段信息"

#~ msgid ""
#~ "Malicous software can damage your data and take control of your system.\n"
#~ "\n"
#~ "The packages below are not authenticated and could therefor be of "
#~ "malicous nature."
#~ msgstr ""
#~ "恶意软件可能破坏您的数据和控制您的系统。\n"
#~ "\n"
#~ "下列软件包没有认证所以实质上可能是恶意软件。"

#~ msgid "A later version is already installed"
#~ msgstr "已经安装了较新的版本"

#~ msgid "Cannot install '%s'"
#~ msgstr "无法安装 '%s'"

#~ msgid "Failed to satisfy all dependencies (broken cache)"
#~ msgstr "满足所有依赖关系失败(缓存损坏)"

#~ msgid "IOError during filelist read: %s"
#~ msgstr "读取文件列表时发生 IO 错误： %s"

#~ msgid "List of files could not be read, please report this as a bug"
#~ msgstr "无法读取列出的文件，请将这作为错误提交"

#~ msgid "Breaks exisiting package '%s' conflict: %s (%s %s)\n"
#~ msgstr "残缺：软件包 '%s' 与 %s (%s %s) 冲突\n"

#~ msgid "Breaks exisiting package '%s' dependency %s (%s %s)\n"
#~ msgstr "残缺：软件包 '%s' 依赖 %s (%s %s)\n"

#~ msgid "_Open..."
#~ msgstr "打开(_O)..."
